cognite/client/_api/files.py

```diff
@@ -685,7 +692,7 @@ def upload_bytes(
     ) -> FileMetadata:
         """Upload bytes or string.
 
-        You can also pass a file handle to content.
+        You can also pass a file handle to 'content'. The file should be opened in binary mode.
```

Comment:
`The file should be opened in binary mode.` I think we don't need to mention this. After all, any content form (str, bytes, TextIO, BinaryIO) is accepted

Reply:
I disagree, `requests` will stop supporting this in the next major release:

<img width="741" alt="image" src="https://github.com/user-attachments/assets/b5f844e4-b4f9-47b1-97da-4038ed451ef9">